core/src/test/java/com/datastax/oss/driver/internal/core/cql/RequestHandlerTestHarness.java

```diff
@@ -148,6 +149,8 @@ protected RequestHandlerTestHarness(Builder builder) {
         .thenReturn(CompletableFuture.completedFuture(null));
 
     Mockito.when(session.getMetricUpdater()).thenReturn(sessionMetricUpdater);
+    Mockito.when(sessionMetricUpdater.isEnabled(any(SessionMetric.class), anyString()))
+        .thenReturn(true);
 
```

Comment:
didn't we miss a test when that ```NoopRequestTracker``` is used as a requestTracker?

Reply:
I added `CqlRequestHandlerTrackerTest`.